include/struct_decls/playtime_decl.h

```diff
@@ -0,0 +1,6 @@
+#ifndef POKEPLATINUM_STRUCT_PLAYTIME_DECL_H
+#define POKEPLATINUM_STRUCT_PLAYTIME_DECL_H
+
+typedef struct Playtime Playtime;
```

Comment:
You can prolly merge this into playtime.h

Reply:
also thoughts on PlayTime over Playtime? would also do playTime over playtime for variable names

Reply:
+1 on `PlayTime`